config/kubernetes-sigs/org.yaml

```diff
@@ -1165,6 +1165,7 @@ teams:
     - msau42
     - saad-ali
     - saikat-royc
+    - pwschuurman
```

Comment:
This list needs to be in lexical order. Can you please make this change?

Reply:
Thanks. Done.